### PR TITLE
1.20.1 compatability for clients

### PR DESCRIPTION
**Pull request by S0lux**

### CODE REVIEW

**Comment by SeraphJACK** [comment]
Thanks for the PR!

I'll test it out soon

File: src/main/java/top/seraphjack/simplelogin/client/EventHandler.java [hunk 1]
```diff
@@ -12,7 +13,7 @@ public final class EventHandler {
 
     @SubscribeEvent
     public static void onGuiOpen(ScreenEvent.Opening event) {
-        if (!(event.getScreen() instanceof SetPasswordScreen) && !PasswordHolder.instance().initialized()) {
+        if (!(event.getScreen() instanceof SetPasswordScreen) && !PasswordHolder.instance().initialized() && event.getScreen() instanceof JoinMultiplayerScreen) {
```
Comment (SeraphJACK) [inline src/main/java/top/seraphjack/simplelogin/client/EventHandler.java]: Checkout #36.

TLDR: Some packs will skip `JoinMultiplayerScreen` and let the player connect to the server directly, so we'd better let the player set their password as soon as the game loads.

Comment (S0lux) [reply src/main/java/top/seraphjack/simplelogin/client/EventHandler.java]: Thanks for pointing that out, I wasn't aware of that potential issue.
